pkg/controller/statusmanager/pod_status.go

```diff
@@ -156,7 +227,7 @@ func (status *StatusManager) SetFromPods() {
 			depProgressing = true
 			// Check for any pods in CrashLoopBackOff state and mark the operator as degraded if so.
 			if !isNonCritical(dep) {
-				hung = append(hung, status.CheckCrashLoopBackOffPods(depName, dep.Spec.Selector.MatchLabels, "Deployment")...)
+				hung = append(hung, status.CheckCrashLoopBackOffPods(depName, dep.Spec.Selector.MatchLabels, "Deployment", cnoclient.DefaultClusterName)...)
```

Comment:
To make it more future proof and generic we should use the `clusterName` here and pick the `DefaultClusterName` if `clusterName` is empty

Reply:
will fix it when we use obj that supports clusterName.

Reply:
Fixed.